_includes/meta.html

```diff
@@ -22,13 +28,13 @@
 
 <!-- Description -->
 {% if page.description %}
-  <meta name="description" content="{{ page.description | xml_escape }}">
-  <meta property="og:description" content="{{ page.description | xml_escape }}" />
-  <meta property="twitter:description" content="{{ page.description | xml_escape }}" />
+  <meta name="description" content="{% t page.description %}">
```

Comment:
I guess you can't run `xml_escape` directly on the liquid translation tag so I had to work around it using `capture`.